tensorboard/webapp/widgets/card_fob/card_fob_controller_component.scss

```diff
@@ -54,3 +55,13 @@ limitations under the License.
   opacity: 0.8;
   cursor: pointer;
 }
+
+.horizontal-prospective-area {
+  height: 25px;
+  transform: translateY(225px);
```

Comment:
These values seem fairly arbitrary, especially the translate. Given the feature isn't fully implemented I can't test this to verify it works with different screen sizes. Can you please leave a comment explaining this?

Reply:
Thanks for checking this. Yes I agree this is not a good css style. Updated to fit x-axis.